tests/vcs/git/GitCommitBearTest.py

```diff
@@ -269,6 +269,24 @@ def test_body_checks(self):
                          [])
         self.assertTrue(self.msg_queue.empty())
 
+        # Testing body_regex
+        self.git_commit('Shortlog\n\n'
+                        'First line, blablablablablabla.\n'
+                        'Another line, blablablablablabla.\n'
+                        'Fix 1112')
+        self.assertEqual(self.run_uut(
+                            body_regex=r'Fix\s+[1-9][0-9]*\s*'), [])
+
+        # Matching with regexp, no match found
+        self.git_commit('Shortlog\n\n'
+                        'First line, blablablablablabla.\n'
+                        'Another line, blablablablablabla.\n'
+                        'Closes #01112')
+        self.assertEqual(self.run_uut(
+                            body_regex=r'Closes\s+#[1-9][0-9]*\s*'),
```

Comment:
4 spaces indentation relative to opening expression/statement, same for the others above :)